### ISSUE TITLE
Segmentaion fault with pybind11::error_already_set

### ISSUE THREAD

**Issue by jayapr** [CiscoDevNet/ydk-gen#1074]
Issue tracker is ONLY used for reporting bugs. Please use the [YDK Community](https://communities.cisco.com/community/developer/ydk) for any support issues.

## Expected Behavior
Run gets interrupted when we  instantiate non-top-level containers!

## Current Behavior
Aborted (core dumped) Segmentaion fault with pybind11::error_already_set

## Your Script

```

```

## Logs
Enable [logging](http://ydk.cisco.com/py/docs/guides/introduction.html#logging) and post the logs below

```
terminate called after throwing an instance of 'pybind11::error_already_set'
  what():  TypeError: '>=' not supported between instances of 'int' and 'NoneType'

At:
  /usr/cisco/packages/python/python-3.6.7/lib/python3.6/logging/__init__.py(1548): isEnabledFor
  /usr/cisco/packages/python/python-3.6.7/lib/python3.6/logging/__init__.py(1307): info

Aborted (core dumped)
```

**Comment by ygorelik** [comment]
This issue might be potentially fixed with #1050, which is yet to be tested.

**Comment by ygorelik** [comment]
Could not reproduce the issue. Hence closing.
